### PR TITLE
Add for-each to C++ loops

### PR DESCRIPTION
**Pull request by connorpark24**
### Description

I have added an explanation for for-each loops in C++ to iterate over a given container. I have provided both an explanation and brief example of the topic. The syntax for a traditional for loop can be confusing/overwhelming to beginners, so providing the syntax for the for-each loop may contribute to a better understanding of the concept of a loop in C++.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by helfese — APPROVED** [review]

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@connorpark24 Please read the comments carefully and make all the suggested changes. Let me know once done.

File: content/cpp/concepts/loops/loops.md [hunk 1]
```diff
@@ -43,3 +44,18 @@ for (int i = 0; i < 10; i++) {
...
+
+The example below prints 1 to 5 on the screen:
+
+> **Note:** The variable is a copy of the object in the vector. To act on the object in the vector, declare the variable by reference (&num).
+
+```cpp
+vector<int> container = {1, 2, 3, 4, 5};
+for (int num : container) {
+  std::cout << num << "\n";
+}
```
[the remark this answers is not in the document]
Comment (connorpark24) [reply content/cpp/concepts/loops/loops.md:60]: Just updated the docs! Let me know if I need to change anything. This is my first time submitting a PR on a project so I am open to any feedback.

File: content/cpp/concepts/loops/loops.md [hunk 2]
```diff
@@ -43,3 +44,28 @@ for (int i = 0; i < 10; i++) {
...
+
+The example below uses `for-each` loop to print 1 to 5 from a vector `container`:
+
+> **Note:** The variable is a copy of the object in the vector. To act on the object in the vector, declare the variable by reference (&num).
+
+```cpp
+vector<int> container = {1, 2, 3, 4, 5};
+for (int num : container) {
+  std::cout << num << "\n";
+}
```
Comment (ishg-153) [inline content/cpp/concepts/loops/loops.md:60]: You need to copy and paste the entire code in your file that gives you the following output that you've now added.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@connorpark24 Resuggested the one change you missed from the last review.

**Comment by ishg-153** [comment]
@connorpark24 It's been over a week are you still working on this PR.

**Pull request closed without merging**